br/pkg/restore/import.go

```diff
@@ -719,7 +752,32 @@ func (importer *FileImporter) downloadSST(
 	for _, p := range regionInfo.Region.GetPeers() {
 		peer := p
 		eg.Go(func() error {
-			resp, err := importer.importClient.DownloadSST(ectx, peer.GetStoreId(), req)
+			importer.storeWorkerPoolRWLock.RLock()
+			workerCh := importer.storeWorkerPoolMap[peer.GetStoreId()]
```

Comment:
Should we consider add/remove store during restore?

Reply:
let's handle this in another PR